tests/foreman/ui/test_environment.py

```diff
@@ -34,17 +36,13 @@ def test_create_env_positive_1(self, name):
 
         """
         with Session(self.browser) as session:
-            make_env(session, name=name)
-            search = self.environment.search(name)
-            self.assertIsNotNone(search)
-
-    @data(
-        gen_string('alpha', 255),
-        gen_string('numeric', 255),
-        gen_string('alphanumeric', 255)
-    )
```

Comment:
I am just thinking if 255 characters is a valid value for environment entity, so maybe, we can move to our typical valid_data_list except all data types that are not 'alpha', 'alphanumeric' and 'numeric' or at least specify valid_env_names() as gen_string of 255 characters, so we can remove `test_create_env_positive_2`?